tle/cogs/hard75Challenge.py

```diff
@@ -139,20 +84,20 @@ async def leaderboard(self,ctx):
         _PER_PAGE = 10
 
         def make_page(chunk, page_num):
-            style = table.Style('{:>}  {:<}  {:>}  {:>}')
+            style = table.Style('{:>}  {:<}  {:>} {:>}')
             t = table.Table(style)
-            t += table.Header('#', 'Name', 'Longest streak', 'Current Streak')
+            t += table.Header('#', 'Name', 'Longest', 'Current')
```

Comment:
you want to go ahead with these or we should come up with a different idea?

Reply:
I like them for now.